docs/installation.md

```diff
@@ -75,94 +75,33 @@ kubectl config use-context admin-ctx
-
-Installed helm version can be obtained by using the following command:
+Verify helm is installed and helm repo is updated. See [helm docs](https://helm.sh/docs/intro/install/#from-script) for setting up helm v3. Installed helm version can be obtained by using the following command:
 
 ```
 helm version
 ```
 Example output:
 
 <div class="co">
-Client: &version.Version{SemVer:"v2.16.8", GitCommit:"145206680c1d5c28e3fcf30d6f596f0ba84fcb47", GitTreeState:"clean"}
-Server: &version.Version{SemVer:"v2.16.8", GitCommit:"145206680c1d5c28e3fcf30d6f596f0ba84fcb47", GitTreeState:"clean"}
+version.BuildInfo{Version:"v3.6.0", GitCommit:"7f2df6467771a75f5646b7f12afb408590ed1755", GitTreeState:"clean", GoVersion:"go1.16.3"}
 </div>
 
-
-In the **default installation mode**, use the following command to install the latest version of OpenEBS in `openebs` namespace.
-
+Setup helm repository
 ```
 helm repo add openebs https://openebs.github.io/charts
 helm repo update
-helm install --namespace openebs --name openebs openebs/openebs
 ```
 
-**For Helm v3** 
-
-See [helm docs](https://helm.sh/docs/intro/install/#from-script) for setting up helm v3. Installed helm version can be obtained by using the following command:
-
-```
-helm version
-```
-Example output:
-
-<div class="co">
-version.BuildInfo{Version:"v3.0.2", GitCommit:"19e47ee3283ae98139d98460de796c1be1e3975f", GitTreeState:"clean", GoVersion:"go1.13.5"}
-</div>
-
-OpenEBS installation with helm v3 can be done by 2 ways:
-
-**Option 1:** Helm v3 takes the current namespace from the local kube config and use that namespace the next time the user executes helm commands. If it is not present, the default namespace is used. Assign the `openebs` namespace to the current context and run the following commands to install openebs in `openebs` namespace. 
-
-To view current context, run the following:
-```
-kubectl config current-context
-```
-Assign `openebs` namespace to the current context:
-```
-kubectl config set-context <current_context_name> --namespace=openebs
+Install OpenEBS helm chart with default values
 ```
-Create OpenEBS namespace
-```
-kubectl create ns openebs
-```
-Install openebs with chart name as `openebs`:
-```
-helm repo add openebs https://openebs.github.io/charts
-helm repo update
-helm install openebs openebs/openebs
-```
-To view the chart
-```
-helm ls
+helm install openebs --namespace openebs openebs/openebs --create-namespace
 ```
 The above commands will install OpenEBS in `openebs` namespace and chart name as `openebs` 
 
-**Option 2:** By mentioning namespace in helm command
-
-Create OpenEBS namespace
-```
-kubectl create ns openebs
-```
-Install openebs with chart name as `openebs`:
-```
-helm repo add openebs https://openebs.github.io/charts
-helm repo update
-helm install --namespace openebs openebs openebs/openebs
-```
 To view the chart
 ```
 helm ls -n openebs
 ```
-The above commands will install OpenEBS in `openebs` namespace and chart name as `openebs` 
 
 
 **Note:** 
```

Comment:
hyperlink associated to here is not going to correct one. it is pointing to the same page only.